Update README with a wiki entry - fixes #6

rustberry opened the pull request:
Added instructions on connecting to rpi remotely on our wiki page.
- this fixes #6

rustberry committed:
Update README with a wiki entry

Added instructions on connecting to rpi remotely on our wiki page.

omarcostahamido commented on README.md:
```diff
@@ -153,6 +153,8 @@ Given the objective of making this project available to creative artists, the co
 - https://www.geeksforgeeks.org/vlc-module-in-python-an-introduction/
 - [forum thread: Rpi4: YouTube Full HD 1080p is slow! How to fix?](https://forums.raspberrypi.com/viewtopic.php?t=302787)
 
+Also don't forget to checkout our [wiki](https://github.com/omarcostahamido/PVM/wiki)! It contains instructions on various topics like [setting up remote access to the Raspberry Pis](https://github.com/omarcostahamido/PVM/wiki/How-to-connect-to-a-Raspberry-Pi-remotely) and how to run the examples.
```
- Though it is ok to have a general call for attention to the wiki at the bottom, this doesn't look like the place to refer to specific entries like it is right now.
- please move `setting up remote access to the Raspberry Pis` reference up to where it makes sense from the perspective of someone who's following along this step-by-step. I would say somewhere between `### Raspberry Pi OS` and `### Set up environment`
- it now also feels that `### Raspberry Pi OS` shall have a quick note at the end mentioning that the first time you boot up the rpi device after loading the flashed sd card it probably needs to be connected to a screen and keyboard and mouse to setup the user and password, or no? Can't remember this step.
- since you're working on readme updates please delete the empty sub-section `### Prerequisites` line 19
- please delete also line 145 which is a duplicate (my mistake)

rustberry replied:
> please move setting up remote access to the Raspberry Pis reference up to where it makes sense from the perspective of someone who's following along this step-by-step. I would say somewhere between ### Raspberry Pi OS and ### Set up environment

Agree, that's the better place.

> it now also feels that ### Raspberry Pi OS shall have a quick note at the end mentioning that the first time you boot up the rpi device after loading the flashed sd card it probably needs to be connected to a screen and keyboard and mouse to setup the user and password, or no? Can't remember this step.

Yes, people do need to connect rpi to a screen and keyboard and mouse for at least once. I just realized that https://www.raspberrypi.com/ also has very good guides on installing the OS and first boot configuration. I'll add the link for reference.

rustberry replied:
Done. Please have a look when you have time 😃 @omarcostahamido. I also added a `.gitattributes` file to make the line endings consistent when the repository is receiving commits from collaborators from different operating systems (e.g. Windows and Mac).

omarcostahamido replied:
@rustberry  I'm going to try to add a commit to the branch with some re-articulations. Easier than to just try to explain here. Thank you

omarcostahamido reviewed (changes requested):
Please see my comments on the review below.
Please also change your OP to include a line that reads: `- this fixes #6 `
Thank you.

rustberry committed:
Add changes to README and add .gitattributes

omarcostahamido committed:
rearticulations on the readme

- remove ending on `### Raspberry Pi OS`
- re-articulate `### Connect to a Raspberry Pi`
- add extra blank line before the start of each new section/sub-section
- remove deprecated references to `pmv_alt.py`
- removed irrelevant known limitations referring to v0.2.1

omarcostahamido reviewed (approved):
LGTM

omarcostahamido commented:
I've updated the OP with the trigger words that I've requested [here](https://github.com/omarcostahamido/PVM/pull/27#pullrequestreview-1161110532): `- this fixes #6`

rustberry commented:
Thanks, LGTM.
